extensions/m_webirc.c

```diff
@@ -121,6 +123,26 @@ mr_webirc(struct Client *client_p, struct Client *source_p, int parc, const char
 		return 0;
 	}
 
+	if (parc >= 6)
+	{
+		char *s;
+		for (s = strtok(parv[5], " "); s != NULL; s = strtok(NULL, " "))
+		{
+			if (!ircncmp(s, "secure", 6) && (s[6] == '=' || s[6] == '\0'))
+				secure = 1;
+		}
+	}
+
+	if (secure && !IsSSL(source_p))
+	{
+		sendto_one(source_p, "NOTICE * :CGI:IRC is not connected securely; marking you as insecure");
+		return 0;
```

Comment:
Hrm, this returns without processing `WEBIRC`. Is that intended? (The notice just says "marking you as insecure".)

Reply:
oops!